### DISCUSSION TITLE
Error when plotting it says kernel crashed

### DISCUSSION THREAD

**Discussion by Harshith-reddy-td** [mrdbourke/pytorch-deep-learning#941] [Q&A]
code: 
# Plot the training and testing data 
def plot_predictions(train_data = X_train,
                 train_labels = y_train,
                 test_data = X_test,
                 test_labels = y_test,
                 predictions = None):
  plt.figure(figsize = (10,7))
  plt.scatter(train_data,train_labels,c = 'b',s = 4,label = "Training data")
  plt.scatter(test_data,test_labels,c = 'g',s = 4,label = "Test data")

  if predictions is not None:
    plt.scatter(test_data,predictions,c = 'r',s = 4,label = "Predictions")
  plt.legend(prop = {"size" : 14})
plot_predictions()

result:
The Kernel crashed while executing code in the current cell or a previous cell. 
Please review the code in the cell(s) to identify a possible cause of the failure. 
Click [here](https://aka.ms/vscodeJupyterKernelCrash) for more info. 
View Jupyter [log](command:jupyter.viewOutput) for further details.
![Screenshot 2024-05-29 182850](https://github.com/mrdbourke/pytorch-deep-learning/assets/107536458/1c675879-d2ab-478a-a3e2-d5411d311289)

**Comment by sametucuncu** [comment]


**Reply by Harshith-reddy-td** [reply]
i have run exact code that was given in the github still i am getting this error there is something wrong with my machine config i guess
